pkg/util/kubeconfig.go

```diff
@@ -0,0 +1,334 @@
+package util
+
+import (
+	"encoding/base64"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"sync/atomic"
+
+	"gopkg.in/yaml.v2"
```

Comment:
is there a reason to use this over sigs.k8s.io/yaml ?

Reply:
Looking at it, I don't think there is. It was the natural path I followed, as I am still getting familiar with client-go.

Reply:
Done!